scripts/unset_autokey_env_vars.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
```

Comment:
Maybe consider pushd and pops to handle environment variables.  Then, don't need addition code to cleanup.

Reply:
As we discussed in our call yesterday, pushd and pops may not be helpful here since we are setting environment variables and not creating/destroy directories.